site/gdocs/components/HomepageIntro.scss

```diff
@@ -183,7 +183,10 @@
         display: block;
 
         @include sm-only {
-            height: 172px;
+            // All our new featured images are published at 1200px x 630px
+            // Older featured images aren't published at this size, so we need to
+            // crop them to the correct aspect ratio so that they're all flush
+            height: calc(298px / (1200 / 630));
             width: 298px;
             object-fit: cover;
```

Comment:
Did you also consider using `contain`? I think it may look nicer, especially given that many of our thumbnails also contain text that might then be cropped.

Reply:
Yeah I guess the risk with that is that the image won't be flush horizontally:

<img width="298" alt="image" src="https://github.com/owid/owid-grapher/assets/11844404/0c5d2d9d-184b-4197-b70b-0e7390bf0ce3">

I'll run a quick script to see what sorts of aspect ratios we currently have before making the call 🙂

Reply:
I tallied up the aspect ratio of all our featured images and it's almost a perfect split above and below 1200/630

total above: 168
total below: 158

so with `contain` we'd either lose vertical alignment on the wider images, or lose horizontal alignment on the taller images.

but, given that not too many of our thumbnails have text, and going forward all new posts will have 1.9048 aspect ratios, I think `cover` is the best choice.

```
0.6898: 1
0.7627: 1
0.7778: 1
0.7832: 1
0.9177: 1
1: 3
1.0355: 1
1.056: 1
1.0957: 1
1.1253: 1
1.1263: 1
1.1996: 1
1.2213: 1
1.2741: 3
1.2841: 1
1.3164: 1
1.3314: 1
1.3323: 1
1.3417: 1
1.3518: 1
1.3521: 1
1.3573: 1
1.3986: 1
1.4031: 1
1.405: 1
1.4101: 1
1.411: 2
1.4145: 1
1.4167: 43
1.4168: 2
1.4182: 1
1.4358: 1
1.4409: 1
1.4464: 1
1.4542: 1
1.46: 1
1.4627: 1
1.4771: 1
1.4917: 1
1.5: 1
1.5275: 1
1.5417: 1
1.5789: 1
1.5992: 1
1.6094: 1
1.6152: 2
1.6257: 1
1.6622: 1
1.6652: 1
1.6776: 2
1.6841: 1
1.7252: 1
1.7332: 1
1.7339: 1
1.7416: 1
1.7442: 5
1.7574: 2
1.7646: 1
1.7753: 1
1.7778: 1
1.7833: 2
1.7914: 1
1.7926: 1
1.7937: 1
1.7938: 1
1.7952: 1
1.815: 1
1.8185: 1
1.8197: 2
1.8417: 1
1.8434: 1
1.8616: 1
1.8623: 1
1.8693: 1
1.8729: 1
1.8912: 1
1.8922: 1
1.895: 1
1.8987: 1
1.9016: 1
1.9017: 20
1.9018: 1
1.9048: 115
1.905: 1
1.9078: 1
1.9103: 1
1.9106: 35
1.9108: 49
1.911: 1
1.9124: 1
1.9139: 2
1.9171: 1
1.92: 1
1.922: 1
1.9488: 1
1.9722: 1
1.9802: 1
1.9876: 1
2.0041: 1
2.0144: 3
2.0174: 1
2.0197: 1
2.0319: 1
2.0393: 1
2.0546: 1
2.0855: 1
2.087: 1
2.1084: 1
2.1093: 1
2.1327: 1
2.151: 1
2.1637: 1
2.1812: 1
2.2161: 1
2.2339: 1
2.2869: 1
2.2878: 1
2.339: 1
2.4237: 1
2.4395: 1
2.4561: 1
2.4887: 1
2.4992: 1
2.535: 7
2.545: 1
2.5478: 33
2.5482: 1
2.8525: 1
```